scipy/interpolate/_fitpack_impl.py

```diff
@@ -745,6 +745,7 @@ def splder(tck, n=1):
         return splantider(tck, -n)
 
     t, c, k = tck
+    c = c.copy()
```

Comment:
`c` is reassigned a few lines below (`c = (c[1:-1-k]  ...`), and not modified - why is this needed?

Reply:
For some reason, without the copy, values in `c` were mutating even if the function definition was not performing any major operation that caused changes in the values stored.

Reply:
There was a concurrency situation on the corresponding tests, I've just fixed them and removed these copies.